test/e2e/framework/kubelet_stats.go

```diff
@@ -494,10 +494,10 @@ func GetKubeletHeapStats(c clientset.Interface, nodeName string) (string, error)
 	if errRaw != nil {
 		return "", err
 	}
-	kubeletstatsv1alpha1 := string(raw)
+	kubeletstatsv1alpha1str := string(raw)
```

Comment:
what about just `kubeletStats` ? or `heapStats`? it's unusual to put the type in a variable name in go and this is pretty long. 

https://github.com/golang/go/wiki/CodeReviewComments#variable-names

```suggestion
	heapStats := string(raw)
```